README.md

```diff
@@ -223,13 +223,13 @@ export type UserFriendDocument = mongoose.Types.Subdocument & {
   uid: UserDocument["_id"] | UserDocument;
 } & UserFriend;
 
-export type UserDocument = mongoose.Document & {
-  metadata?: any;
-  friends: mongoose.Types.DocumentArray<UserFriendDocument>;
-  city: {};
-  name: any;
-  isMetadataString: (this: any) => boolean;
-} & User;
+export type UserDocument = mongoose.Document &
+  UserMethods & {
+    metadata?: any;
```

Comment:
Is this a "user can put anything in it" kind of `any` type?

Might be worth looking into [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation) in that case, so users can provide their type.

Reply:
The `any` comes from `metadata` being defined as a `Mixed` mongoose type.